packages/form-js-viewer/src/render/components/form-fields/Textarea.js

```diff
@@ -0,0 +1,89 @@
+import { isArray, isObject } from 'min-dash';
+import { useContext } from 'preact/hooks';
+
+import { FormContext } from '../../context';
+
+import Description from '../Description';
+import Errors from '../Errors';
+import Label from '../Label';
+
+import {
+  formFieldClasses,
+  prefixId
+} from '../Util';
+
+const type = 'textarea';
+
+
+export default function Textarea(props) {
+  const {
+    disabled,
+    errors = [],
+    field,
+    value = ''
+  } = props;
+
+  const {
+    description,
+    id,
+    label,
+    validate = {}
+  } = field;
+
+  const { required } = validate;
+
+  const onInput = ({ target }) => {
+
+    props.onChange({
+      field,
+      value: target.value
+    });
+
+    autoSizeTextarea(target);
+
+  };
+
+  const autoSizeTextarea = (textarea) => {
```

Comment:
I assume auto-resize is our general strategy here right? So we accept that it's not configurable (e.g., to have a fixed height only).

Reply:
We can add this as a visual configuration in the future, but for the initial version of this, I don't think it's worth it.